gammapy/modeling/models/spatial.py

```diff
@@ -590,9 +590,10 @@ class DiskSpatialModel(SpatialModel):
     phi : `~astropy.coordinates.Angle`
         Rotation angle :math:`\phi`: of the major semiaxis.
         Increases counter-clockwise from the North direction.
-    edge : `~astropy.coordinates.Angle`
+    edge_width : float
         Width of the edge. The width is defined as the range within the
-        smooth edges of the model drops from 95% to 5% of its amplitude.
+        smooth edge of the model drops from 95% to 5% of its amplitude.
```

Comment:
Thanks @adonath! The only comment I have is that this sentence should probably be re-formulated

Reply:
Or maybe just "within which"